diff --git a/node_modules/@react-native-firebase/firestore/android/.project b/node_modules/@react-native-firebase/firestore/android/.project
new file mode 100644
index 0000000..c08c842
--- /dev/null
+++ b/node_modules/@react-native-firebase/firestore/android/.project
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>react-native-firebase_firestore</name>
+	<comment>Project react-native-firebase_firestore created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1762715184130</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RCTConvert+FIRLoggerLevel.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RCTConvert+FIRLoggerLevel.h
index 7f5d782..2f4edba 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RCTConvert+FIRLoggerLevel.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RCTConvert+FIRLoggerLevel.h
@@ -15,7 +15,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <React/RCTConvert.h>
 
 @interface RCTConvert (FIRLoggerLevel)
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h
index 5df82d5..b819254 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h
@@ -15,7 +15,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <Foundation/Foundation.h>
 #import <RNFBApp/RNFBSharedUtils.h>
 #import <RNFBFirestoreQuery.h>
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.h
index b2cd6d2..3884dc8 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.h
@@ -16,7 +16,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <React/RCTBridgeModule.h>
 
 @interface RNFBFirestoreCommon : NSObject
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreDocumentModule.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreDocumentModule.h
index 8a314bd..4e007ee 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreDocumentModule.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreDocumentModule.h
@@ -15,7 +15,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <Foundation/Foundation.h>
 #import <RNFBApp/RNFBSharedUtils.h>
 #import <React/RCTBridgeModule.h>
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreModule.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreModule.h
index a989534..6b92e95 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreModule.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreModule.h
@@ -15,7 +15,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <Foundation/Foundation.h>
 #import <RNFBApp/RNFBSharedUtils.h>
 #import <React/RCTBridgeModule.h>
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreQuery.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreQuery.h
index 9521900..b513b4b 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreQuery.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreQuery.h
@@ -16,7 +16,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <RNFBFirestoreSerialize.h>
 #import <React/RCTBridgeModule.h>
 
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreSerialize.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreSerialize.h
index 6c601af..264af52 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreSerialize.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreSerialize.h
@@ -16,7 +16,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <React/RCTBridgeModule.h>
 
 @interface RNFBFirestoreSerialize : NSObject
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreTransactionModule.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreTransactionModule.h
index 7fb14bb..404b395 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreTransactionModule.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreTransactionModule.h
@@ -15,7 +15,8 @@
  *
  */
 
-#import <Firebase/Firebase.h>
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseFirestore/FirebaseFirestore.h>
 #import <Foundation/Foundation.h>
 #import <RNFBApp/RNFBSharedUtils.h>
 #import <RNFBFirestoreQuery.h>
