diff --git a/node_modules/react-native-worklets/android/CMakeLists.txt b/node_modules/react-native-worklets/android/CMakeLists.txt
index 7ce6058..abcdef1 100644
--- a/node_modules/react-native-worklets/android/CMakeLists.txt
+++ b/node_modules/react-native-worklets/android/CMakeLists.txt
@@ -55,6 +55,8 @@ find_package(ReactAndroid REQUIRED CONFIG)

 if(${JS_RUNTIME} STREQUAL "hermes")
   find_package(hermes-engine REQUIRED CONFIG)
+elseif(${JS_RUNTIME} STREQUAL "jsc")
+  find_package(jsc-android REQUIRED CONFIG)
 endif()

 add_library(worklets SHARED ${WORKLETS_COMMON_CPP_SOURCES}
@@ -104,5 +106,5 @@ if(${JS_RUNTIME} STREQUAL "hermes")
     target_link_libraries(worklets ReactAndroid::hermestooling)
   endif()
 elseif(${JS_RUNTIME} STREQUAL "jsc")
-  target_link_libraries(worklets ReactAndroid::jsctooling)
+  target_link_libraries(worklets jsc-android::jsc)
 endif()
